### PR TITLE
Implement new attls toggle from new zowe json schema

### PR DESCRIPTION
**Pull request by timgerstel**
## Proposed changes

This PR addresses Issue: [*Link to Github issue within https://github.com/zowe/zss/issues* if any]

This PR depends upon the following PRs:

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Change in a documentation
- [ ] Refactor the code 
- [ ] Chore, repository cleanup, updates the dependencies.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## PR Checklist
Please delete options that are not relevant.
- [ ] If the changes in this PR are meant for the next release / mainline, this PR targets the "staging" branch.
- [ ] My code follows the style guidelines of this project (see: [Contributing guideline](https://github.com/zowe/zss/blob/v2.x/master/CONTRIBUTING.md))
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] video or image is included if visual changes are made
- [ ] Relevant update to CHANGELOG.md
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, or describe a test method below

### CODE REVIEW

**Review by 1000TurquoisePogs — CHANGES_REQUESTED** [review]
I think your conditionals are reversed, you need some `!`. http should exist when attls, and keyring should not. Right?

To be honest, the simpler way to do what you're doing is just to **not change** any `components.zss.tls` decision-making, and instead do

components.zss.tls == (zowe.network.attls || components.zss.launcher.attls) ? false : true

**Comment by 1000TurquoisePogs** [comment]
Also you need a changelog when ready.

**Comment by timgerstel** [comment]
> I think your conditionals are reversed, you need some `!`. http should exist when attls, and keyring should not. Right?
> 
> To be honest, the simpler way to do what you're doing is just to **not change** any `components.zss.tls` decision-making, and instead do
> 
> components.zss.tls == (zowe.network.attls || components.zss.launcher.attls) ? false : true

Using a ternary threw a linter error but i think i accomplished the same result anyway

File: defaults.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ components:
   zss:
     port: 7557
     crossMemoryServerName: ${{ ()=> { if (zowe.environments?.ZWED_privilegedServerName) { return zowe.environments.ZWED_privilegedServerName } else { if (zowe.environments?.ZWES_XMEM_SERVER_NAME) { return zowe.environments.ZWES_XMEM_SERVER_NAME } else { return "ZWESIS_STD" } } }() }}
-    tls: true
+    tls: ${{ ()=> { return !(zowe.network.attls || components.zss.launcher.attls) }() }}
```
Comment (1000TurquoisePogs) [inline defaults.yaml]: its possible for network or launcher to not exist, do:
${{ ()=> { return !(zowe.network?.attls || components.zss.launcher?.attls) }() }}

**Pull request closed without merging**